src/main/java/com/aviva/CarRecommendations/InterestFilter.java

```diff
@@ -0,0 +1,65 @@
+package com.aviva.CarRecommendations;
+
+import com.aviva.DataAccess.BankingDataProcess;
+import com.aviva.Entities.AccountHolder;
+import com.aviva.Entities.Car;
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+public class InterestFilter {
+
+    public JSONObject getBestFiveCars(String accountNumber) {
+        BankingDataProcess bdpInit = new BankingDataProcess();
+        AccountHolder user = bdpInit.getAccountHolder(accountNumber);
+
+        EstimateBudget ebInit = new EstimateBudget();
+        float yearlyBudget = ebInit.calculateYearlyBudget(user);
+        user.setSavings(yearlyBudget);
+        float monthlyBudget = ebInit.calculateMonthlyBudget(user);
+        user.setMonthlyBudget(monthlyBudget);
```

Comment:
the budgeting stuff can also be moved to the constructor of accountholder

Reply:
doesn't make sense to do this atm because sometimes we will initialize an account holder without a credit score and at a later point get their latest so for now we should leave it be